gdpr/purposes/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .default import AbstractPurpose
+
+__all__ = ("AbstractPurpose",)
```

Comment:
někde používáš na all () a někde [] sjednoť to prosím

Reply:
Opraveno